jelares/portfolio/src/main/java/com/google/sps/servlet-data/ExampleComments.java

```diff
@@ -0,0 +1,63 @@
+// Copyright 2020 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package com.google.sps.data;
+
+import java.util.ArrayList;
+import java.util.List;
+
+/** Class containing numComments example comments */
+public final class ExampleComments {
+  private final List<String> comments;
+  private int numComments;
+
+  /**
+   * a mutable list of numComments comments
```

Comment:
Consider using capitalization and punctuation, for easier reading, and as shown in examples:
https://google.github.io/styleguide/javaguide.html#s7-javadoc

Reply:
Updated the documentation